src/components/NoteState/NoteState.js

```diff
@@ -40,7 +36,22 @@ function NoteState(props) {
   function onStateOptionsButtonClick() {
     setIsOpen(false);
   }
-
+  const getStatusIcon = () => {
+    switch (annotationState) {
+      case 'Assessors':
```

Comment:
Can we move these states to an enum?

Reply:
Enum couldn't be used here as the files are js not ts , do you mean try making it something like object or map ? @naueramant

Reply:
ah yeah that makes sense! It's fine as it is then :)